RELEASE_PROCESS.rst

```diff
@@ -75,3 +120,13 @@ Merge changes to develop
      git checkout develop
      git merge master
      git push origin develop
+
+Update the release on github
+----------------------------
+
+You can find the release on github at https://github.com/juju/juju-gui/tag/<the newest tag>. Update the
+release notes with the entry from CHANGES.yaml. Upload the dist/jujugui-*.tar.bz2 package as a binary.
+
```

Comment:
Can we adapt the tool from blues_browser to parse and upload the changelog as well as the binary? Why do we need the binary?